opcua/uatypes.py

```diff
@@ -869,7 +869,7 @@ def from_binary(data):
         return obj
 
     def __str__(self):
-        s = 'DataValue(Value:{})'.format(self.Value)
+        s = 'DataValue(Value:{}'.format(self.Value)
```

Comment:
No, this bracket is not needed here. Just look at the next lines, the closing bracket is added at the end.